sip-core/src/main/java/de/ikor/sip/foundation/core/actuator/routes/AdapterInfoContributor.java

```diff
@@ -32,6 +32,7 @@ public class AdapterInfoContributor implements InfoContributor {
    */
   @Override
   public void contribute(Info.Builder builder) {
+    @SuppressWarnings("unchecked")
```

Comment:
We can not avoid suppressing this warning, but it would be good if we could get closer to there by using 
`public <T> T get(String id, Class<T> type)`
so that the casting is not completely blind.